### ISSUE TITLE
Issues on latest version

### ISSUE THREAD

**Issue by aguinane** [MikeJMcGuire/hass-actron#8]
Hi Mike, Thanks for making this add-on!

I only recently set this up for the first time with v2022.1.2 and everything worked great!
After updating to v2022.1.3 (and v2022.1.4 didn't fix it either) the values have all been stale and controls no longer work.

The only things in the log that look like they might be abnormal are:

```
01-02-2022 09:22:52.44 Que.ProcessPartialStatus() [0x00000003] Unable to read state information: RemoteZoneInfo[3].Sensors[0].Battery_pc
01-02-2022 09:22:52.44 Que.ProcessPartialStatus() [0x00000003] Unable to read state information: RemoteZoneInfo[4].Sensors[0].Battery_pc
01-02-2022 09:50:59.94 Que.GetAirConditionerEvents() [0x0000000C] Responded (Encoding N/A, N/A bytes)
```

All the other logs seem okay. I have a NEO device.

I tried uninstalling the add-on and re-installing and it picked up the air-con fine according to the logs - but does not show up as a new device and entities in MQTT.

Any ideas?

**Comment by MikeJMcGuire** [comment]
Yep, can you capture all of the logs from startup - enough to show that it's received data and processed it. Thx

**Comment by aguinane** [comment]
Hmm can't seem to work out how to get it to show more than last 50 or so rows.

Anyways, I found this error which I think could be the problem:

```
01-02-2022 11:34:57.91 Que.GetAirConditionerEvents() [0x00000003] Error (System.Collections.Generic.KeyNotFoundException): Unable to process API HTTP response. The given key '8' was not present in the dictionary.
```

**Comment by MikeJMcGuire** [comment]
Thanks – can you send the log entries surrounding that line?
